//= error

fn main() {
    std::call("unknown fct");
}